full_snapshot.proto

```diff
@@ -426,6 +426,8 @@ message RelationInformation {
   repeated int32 partition_columns = 19; // list of either column index (when corresponding partition field is a column) or 0 (when expression)
   string partitioned_by = 20;
 
+  NullString toast_table = 21;
```

Comment:
Despite this being a string, maybe `toast_table_name` would be best here, for clarity? (since one could also assume its the OID)

Reply:
I'd rather avoid such a long attribute name. I'm fine with `toast_table` or `toast_name` but `toast_table_name` seems needlessly precise.

If we see a need to store the OID in the future, `toast_name` might be a better fit because it could be paired with `toast_oid`.

Reply:
Sure, toast_name sounds good.